### DISCUSSION TITLE
Carbon ion

### DISCUSSION THREAD

**Discussion by GGadesuda** [e0404/matRad#489] [Q&A]
For carbon ion planning in matRad, area from source to target is defined of air or vacuum?

**Comment by wahln** [comment]
The Generic carbon base data was generated with MC simulations in vacuum (@AdmiralHomes or @becker89 should be able to confirm this from their experiments). So you may see slight range offsets, which we don't consider relevant since the generic base data does not represent a "real" machine anyways. For custom base data, the base data format allows for a WEPL "offset" field to correct for things like nozzle-to-isocenter distance & beam-monitoring elements. We are currently working on an implementation that makes an exact air-WEPL correction given nozzle-to-surface distance and information provided in the base data (on dev_MonteCarlo) combined with our efforts to provide a validated Monte Carlo interface.
